Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -419,6 +419,17 @@ void WebProcess::initializeWebProcess(WebProcessCreationParameters&& parameters,
     // Reply immediately so that the identity is available as soon as possible.
     completionHandler(ProcessIdentity { ProcessIdentity::CurrentProcess });
 
+    m_isLockdownModeEnabled = parameters.isLockdownModeEnabled;
+
+#if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
+    if (isLockdownModeEnabled()) {
+#if PLATFORM(IOS_FAMILY)
+        prewarmLogs();
+#endif
+        registerLogHook();
```

Comment:
How concerned are we about logging that may occur after process launch but *before* receiving this initializeWebProcess IPC?

Reply:
This looks a bit risky to me.